website/src/pages/documents/document.page.tsx

```diff
@@ -165,7 +165,10 @@ export const TabSet = ({ doc }: { doc: Document }) => {
         {docData.sources.length > 0 ? (
           <section className={fullWidth}>
             Original document provided courtesy of{" "}
-            <Link href={docData.sources[0]!.link}>{docData.sources[0]!.name}</Link>.
+            <Link href={docData.sources[0]!.link}>
```

Comment:
here instead of using an "!" i think you should be able to check if the docData.sources[0] itself exists (i think you're already doing that by checking the length, but looks like typescript is finicky) since using a "!" can often lead to unexpected behavior